pyproject.toml

```diff
@@ -20,6 +20,7 @@ classifiers = [
 requires-python = "==3.8.*"
 dependencies = [
     "transformers==4.41.2",
+    "peft==0.12.0",
```

Comment:
Please initialize the legal mail for this one, as soon as possible.

Reply:
@quic-jhugo @carlstreeter-quic Please review and approve using `peft==0.12.0` with QEfficient.

Reply:
Actually, hold on.

Reply:
PR80 is requesting peft 0.11.1.  Those reviews are underway.  Is this a necessary update, or is that version OK?

Reply:
And how do these PRs interact?

Reply:
@carlstreeter-quic #80 is now closed. It was an initial commit. We're working on that feature in a separate branch (`refine`). Both this PR and the `refine` branch will use 0.12.0.

Reply:
Removed this as all packages are added in a different PR.